pkgs/development/libraries/libopenblt/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchgit, cmake, libusb1 }:
+
+stdenv.mkDerivation rec {
+
+  pname = "libopenblt";
+  version = "011500";
+
+  src = fetchgit {
+    url = "https://github.com/feaser/openblt.git";
+    rev = "openblt_v${version}";
+    sparseCheckout = [ "Host/Source/" ];
+    sha256 = "sha256-cWdc6Imd6IBqYQRNEafp3bDDQLgdpE90fxSkY+vzhAM=";
+  };
+
+  preConfigure = "cd ./Host/Source/LibOpenBLT/";
+
+  installPhase = ''
+    cd $NIX_BUILD_TOP/openblt/Host/
+    install -D libopenblt.so $out/lib/libopenblt.so
+    cd $NIX_BUILD_TOP/openblt/Host/Source/LibOpenBLT/
```

Comment:
We can give install the full relative path and avoid the cd

Reply:
for the relative paths as above, the cd can be removed